packages/babel-plugin/src/utils/ast.tsx

```diff
@@ -44,7 +44,7 @@ export const getPathOfNode = <TNode extends {}>(
  * @param node
  * @param parentPath
  */
-export const buildCodeFrameError = (error: string, node: t.Node, parentPath: NodePath) => {
+export const buildCodeFrameError = (error: string, node: t.Node, parentPath: NodePath<any>) => {
```

Comment:
Was it giving error somewhere?

Reply:
it was yeah, complaining that a specific path wasnt a subclass of `t.Node` (when it was a `t.Expression`)